imap_processing/codice/codice_l1a.py

```diff
@@ -504,48 +518,30 @@ def process_codice_l1a(file_path: Path, data_version: str) -> xr.Dataset:
         The ``xarray`` dataset containing the science data and supporting metadata.
     """
     # Decom the packets, group data by APID, and sort by time
-    packets = decom_packets(file_path)
-    grouped_data = group_by_apid(packets)
+    xtce_packet_definition = Path(
+        f"{imap_module_directory}/codice/packet_definitions/{constants.PACKET_TO_XTCE_MAPPING[file_path.name]}"
+    )
```

Comment:
This can be saved for a follow-up PR, but you should be able to get to a single XML definition file now as well.

Reply:
Thanks. I have a few TODOs regarding updates to XTCE, I will tackle this then.